inbox/mailsync/backends/imap/generic.py

```diff
@@ -471,6 +471,13 @@ def create_message(self, db_session, acct, folder, msg):
                       existing_imapuid=existing_imapuid.id)
             return None
 
+        # Check if the message is valid. In some scenarios, we get messages with
+        # the date set to 1970 or a `None` body string.
+        # https://sentry.nylas.com/sentry/sync-prod/group/3387/
+        if msg.body is None or msg.internaldate == datetime(1970, 1, 1, 0, 0):
```

Comment:
Hmm true, but I'm not sure we should be trying to sync either of them? I grouped both of these conditions together because it seemed like either one would be a sign something is not right and this message is borked. We could just look at `msg.body is None` though, because that's the only one that causes an exception downstream.